inference-engine/tests/functional/plugin/shared/src/import_export_tests/import_export_base.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "import_export_tests/import_export_base.hpp"
+
+#include <fstream>
+
+namespace LayerTestsDefinitions {
+
+std::string ImportNetworkTestBase::getTestCaseName(testing::TestParamInfo<exportImportNetworkParams> obj) {
+    InferenceEngine::Precision netPrecision;
+    std::string targetDevice;
+    std::map<std::string, std::string> exportConfiguration;
+    std::map<std::string, std::string> importConfiguration;
+    std::tie(netPrecision, targetDevice, exportConfiguration, importConfiguration) = obj.param;
+
+    std::ostringstream result;
+    result << "netPRC=" << netPrecision.name() << "_";
+    result << "targetDevice=" << targetDevice << "_";
+    for (auto const& configItem : exportConfiguration) {
+        result << "_exportConfigItem=" << configItem.first << "_" << configItem.second;
+    }
+    for (auto const& configItem : importConfiguration) {
+        result << "_importConfigItem=" << configItem.first << "_" << configItem.second;
+    }
+    return result.str();
+}
+
+void ImportNetworkTestBase::Run() {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+
+    configuration.insert(exportConfiguration.begin(), exportConfiguration.end());
+    LoadNetwork();
+    Infer();
+    executableNetwork.Export("exported_model.blob");
+
+    const auto& actualOutputs = GetOutputs();
+    auto referenceOutputs = CalculateRefs();
+    Compare(referenceOutputs, actualOutputs);
+
+    for (auto const& configItem : importConfiguration) {
+        configuration[configItem.first] = configItem.second;
+    }
+    std::fstream inputStream("exported_model.blob", std::ios_base::in | std::ios_base::binary);
```

Comment:
So after running tests artifacts are present on the system? Should we remove this file once test has been successfully executed?

Reply:
I've replaced `fstream` with `stringstream` (as it's done for other tests) so now there are now artifacts creating.